tools/pyvmi/module-list.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+"""module-list.py.
+
+Usage:
+  module-list.py [options] <vm_name>
+
+Options:
+  -h --help     Show this screen.
+"""
+
+import logging
+import sys
+from docopt import docopt
+from contextlib import contextmanager
+
+from libvmi import Libvmi, VMIOS, PageMode
+
+
+def init_logger():
+    logger = logging.getLogger()
+    logger.addHandler(logging.StreamHandler())
+    logger.setLevel(logging.INFO)
+
+
+@contextmanager
+def pause(vmi):
+    vmi.pause_vm()
+    yield
```

Comment:
On error here, the VM will be left paused. Is it intended?

If not the common `contextmanager` pattern is:

```python
@contextmanager
def pause(vmi):
    vmi.pause_vm()
    try:
        yield
    finally:
        vmi.resume_vm()
```

Reply:
I thought the contextmanager was already dealing with this try catch.
thanks, i will add it.